docs/rancherctl/service/index.md

```diff
@@ -59,3 +60,17 @@ $ sudo rancherctl service list
 disabled ubuntu-console
 ```
 
+### Delete
+---
+The `delete` command can be used to delete services that were added dynamically. i.e. All the services that were not compiled in can be deleted. 
+
+```
+$ sudo rancherctl service add etcd.yml
+$ sudo rancherctl service list
+enabled  etcd.yml
+disabled ubuntu-console
+$ sudo rancherctl delete etcd.yml
+$ sudo rancherctl service list
+disabled ubuntu-console
+```
+Note that you **cannot** delete the ubuntu-console , because that was compiled into RancherOS
```

Comment:
Yes, you can remove what is in os-services repo.  We don't actually remove the service, just the reference too it in rancher.yml